cmake/inputs/patches/ep_gcssdk/build.patch

```diff
@@ -179,28 +180,30 @@ index 6144037bc..000000000
 -        LOG_INSTALL ON)
 -endif ()
 diff --git a/super/external/google-cloud-cpp-common.cmake b/super/external/google-cloud-cpp-common.cmake
-index 3d904d5df..60ad3b856 100644
+index 3d904d5df..8b7dc1d5f 100644
 --- a/super/external/google-cloud-cpp-common.cmake
 +++ b/super/external/google-cloud-cpp-common.cmake
-@@ -52,6 +52,8 @@ if (NOT TARGET google-cloud-cpp-common-project)
+@@ -52,6 +52,9 @@ if (NOT TARGET google-cloud-cpp-common-project)
              -DGOOGLE_CLOUD_CPP_TESTING_UTIL_ENABLE_INSTALL=ON
              -H<SOURCE_DIR>
              -B<BINARY_DIR>
 +            -DCMAKE_CXX_FLAGS=-fPIC
```

Comment:
ideally we should be passing through `CMAKE_POSITION_INDEPENDENT_CODE=ON` which will work on windows (or at least be a no-op)

Reply:
I made a note for this in https://app.clubhouse.io/tiledb-inc/story/1710/gcs-sdk-nested-compile-flags